### DISCUSSION TITLE
Mod breaks audio recording on V2 camera

### DISCUSSION THREAD

**Discussion by petermg** [gtxaspec/wz_mini_hacks#349] [Q&A]
I also [posted this in the ISSUES section](https://github.com/gtxaspec/wz_mini_hacks/issues/347), but I compiled the demo.bin as per the instructions. Everything works except the audio recording. I cannot hear any audio and the recordings don't have any either. I tried this on 2 cameras. I flashed it back to the original unmodified firmware and the audio works again but of course now I can't use RTSP. Anyone else notice this? I used the latest firmware as of this post. [4.9.9.1433](https://s3.us-west-2.amazonaws.com/download.wyzecam.com/firmware/v2/demo_4.9.9.1433.bin.zip) (September 27, 2022)

**Comment by ahamill85** [comment]
I have also experienced this.
